spotinst_sdk2/models/hpc/aws/__init__.py

```diff
@@ -0,0 +1,150 @@
+import json
+
+none = "c6791c1d3bcab7d4179f64f0f4d50f6091bfe7fadaa790ba8e98fc8645e2b576"
```

Comment:
This is a random value. Keep it same as in other model files.

Reply:
chnaged